model/project_parser_db.go

```diff
@@ -54,6 +54,26 @@ func ParserProjectFindOneById(id string) (*ParserProject, error) {
 	return ParserProjectFindOne(ParserProjectById(id))
 }
 
+func ParserProjectFindOneByVersion(id string, version string) (*ParserProject, error) {
```

Comment:
Let's rename ID to projectId, or else this reads like version Id. Can you add a function comment to indicate what happens if version isn't passed in?